dynamics-nav/COPYLINKS-Function--RecordRef-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "COPYLINKS Function (RecordRef)"
+description: "Describes the COPYLINKS function (RecordRef), which copies all the links from a particular record."
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.